### DISCUSSION TITLE
Getting a 404 - not found error...

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [wailsapp/wails#2512] [General]
The app was built with Wails 2.4.0 and is giving:

![Error_NLPP](https://user-images.githubusercontent.com/1826391/226589793-c6a2c96d-0316-4adf-bcd8-6f5206430414.png)

I am wondering how that can happen. I see that 2.4.1 fixes some loading bugs. Do you think the 404 problem could be related?

**Comment by MikeCraig418** [comment]
Is this an existing app that stopped working? Or a new app throwing a 404?

**Reply by Robert-M-Muench** [reply]
That's an existing app, which worked, and I rebuilt it using Wails 2.4.0.

I never saw this behavior, nor do I know how this error can happen. I mean, the HTML stuff is embedded... how can this not be found?

**Reply by leaanthony** [reply]
Fixed in v2.4.1
